cpu/sam0_common/periph/i2c.c

```diff
@@ -11,7 +11,7 @@
  * @{
  * @file
  * @brief       Low-level I2C driver implementation
- * @author      Baptiste Clenet <baptiste.clenet@xsoen.com>
+ * @author      Baptiste Clenet <bapclenet@gmail.com>
```

Comment:
I think this change could be part of a separated PR (that will be fast to review)